torch/_inductor/codecache.py

```diff
@@ -1439,7 +1439,9 @@ def load_with_key(
 
             if (time_saved_ns := compiled_graph._time_taken_ns) is not None:
                 cache_info["time_saved_ns"] = time_saved_ns
-                add_remote_cache_time_saved(time_saved_ns, is_backward)
+                METRICS_CONTEXT.increment(
```

Comment:
nit: It's somewhat more idiomatic in the prevailing codebase to use a function to get the context, as this makes it easier to do things like TLS correctly (and you will need to eventually make `METRICS_CONTEXT` live in TLS the more thread safe we make PT2).